README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Include any prerequisites for running these commands, such as needing to have Homebrew installed or having a Go environment set up.